lib/carrierwave/mongoid.rb

```diff
@@ -97,7 +97,7 @@ def serializable_hash(options=nil)
 
           self.class.uploaders.each do |column, uploader|
             if (!only && !except) || (only && only.include?(column.to_s)) || (except && !except.include?(column.to_s))
-              hash[column.to_s] = _mounter(column.to_sym).uploader.serializable_hash
+              hash[column.to_s] = _mounter(column.to_sym).uploaders[0].serializable_hash
```

Comment:
`Mounter` is now defined in separate file and [supports](https://github.com/carrierwaveuploader/carrierwave/blob/master/lib/carrierwave/mounter.rb#L32) multiple uploaders. My `uploaders[0]` seems to break everything for `carrierwave '< 1'` so I am not sure how to implement in correctly.